cmd/infra/ibmcloud_powervs/create.go

```diff
@@ -599,6 +559,51 @@ func (infra *Infra) setupVpcSubnet(option *CreateInfraOptions, v1 *vpcv1.VpcV1)
 	return nil
 }
 
+func (infra *Infra) createVpcSubnet(option *CreateInfraOptions, v1 *vpcv1.VpcV1) (subnet *vpcv1.Subnet, err error) {
+	vpcIdent := &vpcv1.VPCIdentity{CRN: &infra.VpcCrn}
+	resourceGroupIdent := &vpcv1.ResourceGroupIdentity{ID: &infra.ResourceGroupID}
+	subnetName := fmt.Sprintf("%s-hypershift-vpc-subnet", option.InfraID)
+	ipVersion := "ipv4"
+	zones, _, err := v1.ListRegionZones(&vpcv1.ListRegionZonesOptions{RegionName: &option.VpcRegion})
+	if err != nil {
+		return nil, err
+	}
+
+	addressPrefixL, _, err := v1.ListVPCAddressPrefixes(&vpcv1.ListVPCAddressPrefixesOptions{VPCID: &infra.VpcID})
+	if err != nil {
+		return nil, err
+	}
+
+	for i := 0; i < 3; i++ {
```

Comment:
can we loop through `zones.Zones` instead of hardcoded count 3 here?

Reply:
done.